src/ProductConstructionService/ProductConstructionService.Common/ConfigurationExtension.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Configuration;
+
+namespace ProductConstructionService.Common;
+public static class ConfigurationExtension
+{
+    // In the PCS API, environment is set with the ASPNET_ENVIRONMENT variable
+    // In other services, like the SubscriptionTriggerer, it is set with the DOTNET_ENVIRONMENT variable
```

Comment:
In what way? I added them because I had some trouble when I switched to the ApplicationHostBuilder conf builder, so just a reminder if someone is also getting exceptions

Reply:
But they don't belong in this file